Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
```

Comment:
The inclusion of the `--build.cmd`, `--build.bin`, and other flags in the `dev_server` task seems repetitive. It might be beneficial to group similar build options together or even create a separate script for building, reducing the complexity of the command.